<?xml version="1.0"?>

<launch>
  <?ignore
  <!-- SR300 -->
  <include file="$(find realsense2_camera)/launch/rs_rgbd.launch">
    <arg name="camera" value="camera1"/>
  </include>
  <!-- UR5 -->
  <include file="$(find arm_operation)/launch/ur5_real.launch">
    <arg name="robot_ip" value="192.168.50.11"/>
    <arg name="tool_length" value="0.0"/>
  </include>
  ?>

  <!-- robotiq 2-finger gripper -->
  <node pkg="robotiq_2f_gripper_control" type="Robotiq2FGripperRtuNode.py" name="Robotiq2FGripperRtuNode" args="/dev/ttyGripper" output="screen" required="true"/>
  
  <!-- robotiq gripper control -->
  <node pkg="grasp_suck" type="robotiq_gripper_control" name="robotiq_finger_control_node" output="screen" required="true"/>


  <node pkg="vacuum_conveyor_control" type="arduino_control" name="arduino_control">
    <param name="port" value="/dev/ttyArduino"/>
  </node>



  <node pkg="arm_operation" type="set_pose" name="set_pose" output="screen">
    <param name="file_name" value="$(find arm_operation)/data/wp.txt"/>
  </node>

  <!-- Visual System -->
  <!-- Convert pointcloud from eye coord. to hand coord. -->
  <!--node pkg="visual_system" type="pc_transform" name="pc_transform" output="screen" required="true">
    <param name="verbose" value="true"/>
    <param name="num_thres" value="$(arg empty_thres)"/>
    <remap from="camera/color/image_raw" to="/camera1/color/image_raw"/>
    <remap from="camera/aligned_depth_to_color/image_raw" to="/camera1/aligned_depth_to_color/image_raw"/>
    <remap from="camera/color/camera_info" to="/camera1/color/camera_info"/>
  </node-->

</launch>
